src/components/LazyLog/README.md

```diff
@@ -35,3 +35,29 @@ consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
   <LazyLog extraLines={1} enableSearch text={text} caseInsensitive />
 </div>
 ```
+
+Log viewing using a websocket
+
+```jsx harmony
+const url = 'wss://echo.websocket.org';
+let socket = null;
+
+<div style={{ height: 200, width: 902 }}>
+  <button style={{margin: 5}} onClick={() => socket && socket.send(JSON.stringify({ message: '[taskcluster 2018-11-14 21:08:32.452Z] Worker Group: us-east-1' }))}>ping</button>
+  <LazyLog
+    enableSearch
+    url={url}
+    websocket
+    websocketOptions={{
+      onOpen: (e, sock) => {
+        console.log("opened");
+          socket = sock; sock.send('Socket has been opened!')
+        },
+      formatMessage: (e) => {
+          return JSON.parse(e).message;
+      },
+    }}
+  />
+}
+</div>
+```
```

Comment:
http://localhost:6060 shows:

![Screen Shot 2019-11-08 at 1 38 30 PM](https://user-images.githubusercontent.com/3766511/68501856-2245b900-022d-11ea-9f83-4c5228667411.png)

Reply:
@helfi92 thank for looking at this!

what version of node are you running it under? I was not able to repro this issue with either `10.x` or `13.x`

Reply:
Currently using `v12.13.0`

Reply:
Ahh. seems to be ok on Chrome but throws on FF.

Reply:
ah interesting! let me dig into that!